include/cutlass/epilogue/fusion/sm90_visitor_store_tma_warpspecialized.hpp

```diff
@@ -700,7 +701,10 @@ struct Sm90RowReduction {
       reduction_buffer = nullptr;
     }
     else if constexpr (FinalReduction) {
-      auto [M, N, K, L] = problem_shape;
+      // Optionally append 1s until problem shape is rank-4 in case it is only rank-3 (MNK)
```

Comment:
would you consider to remove this comment? for me the code already conveys this message clearly.